libs/components/data-manager/src/lib/modules/data-manager/data-manager-column-picker/data-manager-column-picker.component.ts

```diff
@@ -50,6 +50,11 @@ export class SkyDataManagerColumnPickerComponent implements OnDestroy, OnInit {
         (col) => col.isSelected
       );
     }
+
+    this.isAnyDisplayedColumnSelected =
+      this.displayedColumnData.find(
```

Comment:
Use `this.displayedColumnData.some()` so you don't have to compare it to `undefined`. Also you can remove the `=== true` unless you anticipate other truthy values for `isSelected`.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
resolved in [new PR](https://github.com/blackbaud/skyux/pull/349). closing this PR.